Use clone-deep only in tests

<!-- Not following the template might result in your issue being closed -->

**_Bug report_**

- [x] I've searched for existing issues
- [x] I've checked documentation: https://docs.frontity.org
- [x] I've checked community forum: https://community.frontity.org

**Describe the expected behavior**

<!-- What should happen? -->
`clone-deep` should not be needed after installing `wp-source` package.

**Describe the observed behavior**

<!-- What happens instead? -->
 `clone-deep` is being used in the package's code.

**Describe the steps involved to reproduce the problem**

<!-- Start from the `npx frontity create` command -->
Run `npx frontity create` and then `npx frontity dev` inside the project folder. The following error should appear:
```
ERROR in ./node_modules/@frontity/wp-source/src/index.ts
    Module not found: Error: Can't resolve 'clone-deep'...
```

**Add info about your system**

<!-- Run `npx frontity info` in your Frontity root folder and paste the result here -->

--- a/packages/wp-source/src/index.ts
+++ b/packages/wp-source/src/index.ts
@@ -1,4 +1,3 @@
-import clone from "clone-deep";
 import WpSource from "../types";
 import state from "./state";
 import actions from "./actions";
@@ -7,7 +6,7 @@ import { Api, populate, routeUtils, responseUtils } from "./libraries";
 const wpSource = (): WpSource => ({
   name: "@frontity/wp-source",
   state: {
-    source: clone(state)
+    source: state
   },
   actions: {
     source: actions

--- a/packages/wp-source/src/libraries/handlers/__tests__/author.tests.ts
+++ b/packages/wp-source/src/libraries/handlers/__tests__/author.tests.ts
@@ -1,4 +1,5 @@
 import { createStore, InitializedStore, observe } from "@frontity/connect";
+import clone from "clone-deep";
 import wpSource from "../../../";
 import WpSource from "../../../../types";
 import Api from "../../api";
@@ -12,7 +13,7 @@ import author1PostsCpt from "./mocks/author/author-1-posts-cpt.json";
 let store: InitializedStore<WpSource>;
 let api: jest.Mocked<Api>;
 beforeEach(() => {
-  store = createStore(wpSource());
+  store = createStore(clone(wpSource()));
   store.actions.source.init();
   api = store.libraries.source.api as jest.Mocked<Api>;
 });

--- a/packages/wp-source/src/libraries/handlers/__tests__/category.tests.ts
+++ b/packages/wp-source/src/libraries/handlers/__tests__/category.tests.ts
@@ -1,4 +1,5 @@
 import { createStore, InitializedStore, observe } from "@frontity/connect";
+import clone from "clone-deep";
 import wpSource from "../../../";
 import WpSource from "../../../../types";
 import Api from "../../api";
@@ -12,7 +13,7 @@ import cat1PostsCpt from "./mocks/category/cat-1-posts-cpt.json";
 let store: InitializedStore<WpSource>;
 let api: jest.Mocked<Api>;
 beforeEach(() => {
-  store = createStore(wpSource());
+  store = createStore(clone(wpSource()));
   store.actions.source.init();
   api = store.libraries.source.api as jest.Mocked<Api>;
 });

--- a/packages/wp-source/src/libraries/handlers/__tests__/date.tests.ts
+++ b/packages/wp-source/src/libraries/handlers/__tests__/date.tests.ts
@@ -1,4 +1,5 @@
 import { createStore, InitializedStore } from "@frontity/connect";
+import clone from "clone-deep";
 import wpSource from "../../../";
 import WpSource from "../../../../types";
 import Api from "../../api";
@@ -13,7 +14,7 @@ import date_2019_01_01_posts_cpt from "./mocks/date/2019-01-01-posts-cpt.json";
 let store: InitializedStore<WpSource>;
 let api: jest.Mocked<Api>;
 beforeEach(() => {
-  store = createStore(wpSource());
+  store = createStore(clone(wpSource()));
   store.actions.source.init();
   api = store.libraries.source.api as jest.Mocked<Api>;
 });

--- a/packages/wp-source/src/libraries/handlers/__tests__/post-archive.tests.ts
+++ b/packages/wp-source/src/libraries/handlers/__tests__/post-archive.tests.ts
@@ -1,4 +1,5 @@
 import { createStore, InitializedStore } from "@frontity/connect";
+import clone from "clone-deep";
 import wpSource from "../../../";
 import WpSource from "../../../../types";
 import Api from "../../api";
@@ -11,7 +12,7 @@ import postsCpt from "./mocks/post-archive/posts-cpt.json";
 let store: InitializedStore<WpSource>;
 let api: jest.Mocked<Api>;
 beforeEach(() => {
-  store = createStore(wpSource());
+  store = createStore(clone(wpSource()));
   store.actions.source.init();
   api = store.libraries.source.api as jest.Mocked<Api>;
 });

--- a/packages/wp-source/src/libraries/handlers/__tests__/post-type.tests.ts
+++ b/packages/wp-source/src/libraries/handlers/__tests__/post-type.tests.ts
@@ -1,4 +1,5 @@
 import { createStore, InitializedStore } from "@frontity/connect";
+import clone from "clone-deep";
 import wpSource from "../../../";
 import WpSource from "../../../../types";
 import Api from "../../api";
@@ -12,7 +13,7 @@ import cpt11 from "./mocks/post-type/cpt-11.json";
 let store: InitializedStore<WpSource>;
 let api: jest.Mocked<Api>;
 beforeEach(() => {
-  store = createStore(wpSource());
+  store = createStore(clone(wpSource()));
   store.actions.source.init();
   api = store.libraries.source.api as jest.Mocked<Api>;
 });

--- a/packages/wp-source/src/libraries/handlers/__tests__/tag.tests.ts
+++ b/packages/wp-source/src/libraries/handlers/__tests__/tag.tests.ts
@@ -1,4 +1,5 @@
 import { createStore, InitializedStore, observe } from "@frontity/connect";
+import clone from "clone-deep";
 import wpSource from "../../../";
 import WpSource from "../../../../types";
 import Api from "../../api";
@@ -12,7 +13,7 @@ import tag1PostsCpt from "./mocks/tag/tag-1-posts-cpt.json";
 let store: InitializedStore<WpSource>;
 let api: jest.Mocked<Api>;
 beforeEach(() => {
-  store = createStore(wpSource());
+  store = createStore(clone(wpSource()));
   store.actions.source.init();
   api = store.libraries.source.api as jest.Mocked<Api>;
 });

